app/models/host/discovered.rb

```diff
@@ -146,18 +146,13 @@ def reboot
     proxy_url = self.proxy_url
 
     if proxy_url[:type] == 'proxy'
-      status = ForemanDiscovery::ProxyOperations.new(:url => proxy_url[:url], :operation => 'reboot').parse_put_operation
+      ForemanDiscovery::ProxyOperations.new(:url => proxy_url[:url], :operation => 'reboot').parse_put_operation
     else
-      status = ::ProxyAPI::BMC.new(:url => "http://#{self.ip}:8443").power :action => "cycle"
+      ::ProxyAPI::BMC.new(:url => "http://#{self.ip}:8443").power :action => "cycle"
     end
-
-    msg = status ? 'successful' : 'failed'
```

Comment:
Yeah, I don't see any reason to remove error handling that we know could fire.  Just because the image is now known to be in a working state doesn't mean it always will be...